tests/core/test_holders.py

```diff
@@ -178,3 +178,11 @@ def test_set_not_chaged_variable():
     array = np.asarray([2000])
     holder.set_input('2015-01', array)
     assert_equal(simulation.calculate('salary', '2015-01'), array)
+
+
+def test_set_input_float_to_int():
```

Comment:
This tests pass without the change on `columns.py` 😅 

(tests in this file don't use `scenarios`)

Reply:
Yep, that's on purpose - documenting at what level the desired behaviour is being blocked.